deftree/svcparse/lexer.go

```diff
@@ -68,7 +68,7 @@ func NewTokenGroup(scn *SvcScanner) *TokenGroup {
 					}
 					twostr := string(two)
 					if len(two) > 1 && two[0] == '/' {
-						str += twostr
+						str += onestr + twostr
```

Comment:
What do you mean by "deserves a comment"? As in I should comment the code, or I should explain it here?